shared/actions/tracker.js

```diff
@@ -694,18 +545,13 @@ function _serverCallMap(
 
       addToIdleResponseQueue(() => {
         // Check if there were any errors in the proofs
-        dispatch({type: Constants.updateProofState, payload: {username}})
-
-        dispatch({type: Constants.identifyFinished, payload: {username}})
-
-        dispatch({
-          type: Constants.markActiveIdentifyUi,
-          payload: {username, active: false},
-        })
+        dispatch(TrackerGen.createUpdateProofState({username}))
```

Comment:
`username` is `undefined` here when kbfs access triggers a tracker popup (don't know where to trace what causes this)

Reply:
so it turns out the old code did this but the reducer would toss out these values. i'm changing the reducer helper to do the same.
*but* we really should change how this works cause how this is handled is pretty racy and bad